neuro_flow/ast.py

```diff
@@ -146,14 +146,19 @@ class Job(ExecUnit, JobBase):
     multi: SimpleOptBoolExpr
 
 
+class NeedsLevel(enum.Enum):
```

Comment:
I do not like this naming, but I'm out of ideas.

Reply:
I think we can just reuse `TaskStatus` enum instead of adding a new one.

Reply:
As discussed, we need a separate enum here. Should I rename it?

Reply:
Up to you. It is a part of private API anyway.